book/website/collaboration/stakeholder-engagement.md

```diff
@@ -0,0 +1,116 @@
+
+### Example resources
+- Turing RAM team [requirements gathering questionnaire example](https://github.com/alan-turing-institute/research-application-management/blob/main/docs/requirements_gathering.md) for understanding different team approaches to creating case studies
+
+### RAM insights
+* Often times different people will use the same term to mean completely different things or for different purposes
+* This exercise can help detangle the nuances of a commonly used term and whether or not there is overlap, and thus provides an opportunity to build shared understanding and goals
+* Requirements gathering is informed by the user interview process from the design and qualitative research worlds
+* The work begins after you gather requirements information, which is the starting point to facilitate a group conversation
+
+(cl-stakeholder-strategy-workshops)=
+## Strategy alignment workshops
+### Key question: How can we bring different ideas across a group of collaborators into a shared strategy?
+
+A group of collaborators will struggle to work together and progress if its members are not aligned on what they are trying to achieve together. 
+Strategy alignment goes deeper than agreeing on day to day work - it asks what the team chooses to prioritise, what they value, what motivates them, and ultimately what real-world impact they hope to see from their work.
+A helpful model to inform this exercise is Simon Sinek's [Golden Circle method](https://www.smartinsights.com/digital-marketing-strategy/online-value-proposition/start-with-why-creating-a-value-proposition-with-the-golden-circle-model/), where the team approaches their work in the following order:
+* Why: Why are we doing what we're doing? 
+    * This is the ultimate 'North Star' or 'vision' of the project - the fundamental, emotive reason the work is being done
+* How: How do we achieve our mission? 
+    * This question concerns the broad focus areas, or 'pillars' that setup up the project to achieve the vision
+* What: What do we actually do?
+    * This question takes the why and the how, and identifies key activities that teams can undertake to achieve their goals
+
+It is important to take these conversations in order, so that strategy alignment can build from the bottom-up.
+
+### Example resources
+- [Further RAM Guidance](https://github.com/alan-turing-institute/research-application-management/blob/main/docs/team_strategy.md) on facilitating Why/How/What conversations
+
+### RAM insights
+* Using this basic infrastructure for the conversation helps provide focus and clarity and can lead to a better outcome
+* By not overstructuring the conversation, you will also allow people the space to express their thoughts freely. We feel this is an important approach as it allows open conversations to happen and things to emerge that otherwise might not
```

Comment:
```suggestion
By not overstructuring the conversation, you will also allow people the space to express their thoughts freely. We feel this is an important approach as it allows open conversations to happen and things to emerge that otherwise might not.
```